src/mutation/DeletePersonMutation.js

```diff
@@ -0,0 +1,34 @@
+import { GraphQLString } from 'graphql';
+import { mutationWithClientMutationId } from 'graphql-relay';
+import { Person } from '../model';
+
+export default mutationWithClientMutationId({
+  name: 'DeletePerson',
+  inputFields: {
+    _id: {
+      type: GraphQLString,
+    },
+  },
+  mutateAndGetPayload: async ({ _id }) => {
+    const person = await Person.findOne({ _id });
+
+    if (person) {
+      return person
+        .remove()
+        .then(() => ({
```

Comment:
avoid then and catch, embrace async await

Reply:
please, explain to me, "person" return a promise right? i not understand why use async/await in this case

Reply:
async/await is es6 way to handle promises

Reply:
https://hackernoon.com/6-reasons-why-javascripts-async-await-blows-promises-away-tutorial-c7ec10518dd9